packages/webapp/src/app/utils/initializeFontFace.ts

```diff
@@ -1,38 +0,0 @@
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
-import { interFontName } from '@sb/webapp-core/theme/font';
-
-import interPrimary from '../../assets/fonts/Inter-Regular.woff';
```

Comment:
I'm thinking about the fonts... now `Inter` is not loaded, right?

Reply:
Font looks the same as before 
<img width="1166" alt="image" src="https://github.com/apptension/saas-boilerplate/assets/116499136/9498d80d-db43-47ba-91e6-79bdbbdfc6a8">

Reply:
![image](https://github.com/apptension/saas-boilerplate/assets/2943077/9db27087-b707-43bd-85ab-efc27cba456a)
Yeah but I think we have incorrect config... Either we should load and use `Inter` or remove the custom font and use standard only

Reply:
Corrected, thanks
<img width="1735" alt="image" src="https://github.com/apptension/saas-boilerplate/assets/116499136/3622837b-bd62-4ab2-b27e-446351d327fe">